pom.xml

```diff
@@ -10,14 +10,13 @@
   <packaging>war</packaging>
 
   <name>mvnwar Maven Webapp</name>
-  <!-- FIXME change it to the project's website -->
   <url>http://www.example.com</url>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <junit.version>4.11</junit.version>
+    <junit.version>4.12</junit.version>
```

Comment:
### org.junit : junit : 4.12
⚠️ **No recommended versions** are available for this component (as of _Feb 25, 2020_)

Threat (of 10) | IQ Policy Name | Violation Details
-- | -- | -- | 
9 | Security-High | **High Risk CVSS Score**. Found security vulnerabilities: [CVE-2016-1000338](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344), [CVE-2016-1000340](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344), [CVE-2016-1000342](url), [CVE-2016-1000343](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344), [CVE-2016-1000344](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344)
3 | Security-Low | **Low Risk CVSS Score**. Found security vulnerability: [CVE-2016-6644](http://localhost:8070/assets/index.html#/vulnerabilities/CVE-2016-1000344).
1 | Architecture-Quality | **Version is old**. Age was 8 years, 5 months and 28 days.